quickstarts/sample-application-quickstart.yaml

```diff
@@ -55,23 +55,31 @@ spec:
       summary:
         success: Your sample app is now built!
         failed: Try the steps again.
-    - title: View the associated Git repo
-      description: |
+    - title: View the associated code
+      description: |-
         ### To view the associated code:
-        1. If you aren't already there, go to the [Topology]{{highlight qs-nav-topology}} page in the navigation.
-        2. The icon on the bottom right quadrant of the **nodejs-sample** deployment represents the Git repo of the associated code.  The icon shown can be for Bitbucket, GitHub, GitLab or generic Git. Click on it to navigate to the associated Git repository.
+        1. In the navigation menu, click [Topology]{{highlight qs-nav-topology}}.
+
+        1. The icon on the bottom right quadrant of the **nodejs-basic** deployment either represents the Git repository of the associated code, OR if the appropriate operators are installed, it will bring up the associated code in your IDE.
+
+        1. If the icon shown is Bitbucket, GitHub, GitLab or generic Git, click it to navigate to the associate Git repository. Otherwise, if the icon shown is CodeReady Workspaces or Eclipse Che, click it to bring up the associated code in your IDE.
+      
       review:
-        instructions: |-
-          #### Verify you can see the code associated with the sample app:
-          Was a new browser tab opened with https://github.com/sclorg/nodejs-ex?
-        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        failedTaskHelp: >- 
+          This task isn’t verified yet. Try the task again.
+        instructions: >-
+          Verify that you can see the code associated with the sample app:
+          
+          - Was a new browser tab opened either to your associated Git repository or to your IDE?
+
       summary:
-        success: You have viewed the associated Git repo.
         failed: Try the steps again.
+        success: You viewed the code associated with the **nodejs-basic** deployment.
+
     - title: View the pod status
       description: |-
         ### To view the pod status:
-        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-sample** deployment has a pod donut imposed on the circle, representing the pod status
+        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-basic** deployment has a pod donut imposed on the circle, representing the pod status
```

Comment:
```suggestion
        1. Click on the browser tab with OpenShift.  Notice that the **nodejs-basic** deployment has a pod donut imposed on the circle, representing the pod status.
```

Reply:
Thanks @jhadvig! updated the reference to **nodejs-sample** and added the period.